.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+### Summary
+
+Describe the changes made in this PR and why they are needed.
+
+### Testing
+
+Describe the testing done for this PR.
+
+### Applications affected
+
+List all known applications (UFS WM, JEDI, SRW, etc.) intentionally or unintentionally affected by this PR.
```

Comment:
How do I know that my PR affects something unintentionally?

Reply:
I kept this language from the JCSDA template because I like that it sets the expectation that PR submitters are responsible for not breaking things that aren't relevant to their application of interest (and for that matter, it might prompt them to think harder about the impacts of their changes).